phys2bids/bids.py

```diff
@@ -242,3 +242,44 @@ def participants_file(outdir, yml, sub):
             participants_data = ['n/a'] * header_length
             participants_data[p_id_idx] = f'sub-{sub}'
             utils.append_list_as_row(file_path, participants_data)
+
+
+def dataset_description_file(outdir):
+    """
+    Create dataset_description.json file if it does not exist.
+    If it exists, do nothing.
+
+    Parameters
+    ----------
+    outdir: path
+        Full path to the output directory.
+
+    """
+    # dictionary that will be written for the basic dataset description version
+    data_dict = {"Name": os.path.splitext(os.path.basename(outdir))[0],
```

Comment:
I would check first if the file exists. If it does, we could read it and update the missing fields and the bids version. If it does not, we could create it.

Reply:
I wouldn't do that, All the fields there are Required except datatype, therefore there is no need to update for the moment. A different thing is that we add new fields in the future

Reply:
Fair enough. Can you move this into the if statement then?

Reply:
Plus we cannot update the bids version of the whole dataset because we are not ensuring bids specs from other kind of data